tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = packaging, py35, py36, py37, py38, check_codestyle, check_isort
+envlist = packaging, py35, py36, py37, py38, py39, check_codestyle, check_isort
```

Comment:
Are these environments actually used if they're not repeated below :thinking:

Reply:
My understanding is that tox uses that envlist and the default `[testenv]` section when invoked without an explicit `-e` parameter on the commandline.

Reply:
Ah indeed, TIL!